lib/xapi-stdext-date/date.ml

```diff
@@ -30,7 +30,9 @@ let rfc822_to_string x = x
 
 (* ==== ISO8601/RFC3339 ==== *)
 
-type iso8601 = Ptime.t
+type iso8601 =
+  | UTC of Ptime.t
+  | Localtime of string
```

Comment:
Can't we convert it to UTC, or store the timezone offset as part of the type instead of making completely a string? https://erratique.ch/software/ptime/doc/Ptime.html#TYPEtz_offset_s

Reply:
See also https://erratique.ch/software/ptime/doc/Ptime_clock.html, `val current_tz_offset_s : unit -> Ptime.tz_offset_s option`